relay/filter.py

```diff
@@ -4,6 +4,7 @@
 
 from .event import NostrEvent
 
+#from loguru import logger
```

Comment:
remove this and other commented lines below related to logging

Reply:
Will do over weekend